index.html

```diff
@@ -0,0 +1,147 @@
+        <div><p class="main">Main Program</p></div>
+        <div class="speaker1m-hr"> </div>
+        <div class="main-program">
+            <div class="program">
+                <img  class= "imglecture" src="img/program_icon_01.png" alt="lecture image">
+                <h3>Lectures</h3>
+                <p class="desc3"> There will lectures on different programming languages both for the front end and back end 
+                    development. 
+                </p>
+            </div>
+            <div class="program"> 
+                <img src="img/program_icon_02.png" alt="Exhibition image">
+                <h3> Exhibition </h3>
+            <P class="desc3i"> The practical exhibition of the topic and it applicaion and usage is will be shown</P>
+            </div>
+            <div class="program">
+                <img src="img/program_icon_03.png" alt="Forum image">
+                <h3>Forum</h3> 
+            <p class="desc3e"> Students will be allowed to interact with the programmers and ask
+                questions. 
+            </p>
+        </div>
+        <div class="program"> 
+            <img src="img/program_icon_04.png" alt="Workshop image">
+            <h3> Workshop</h3>
+            <p class="desc3i">Students will be given some projects to work on which will supervised by programmers</p>
+        </div>
+        <div class="program">
+            <img src="img/program_icon_05.png" alt="Milestone image">
+            <h3>Milestone</h3>
+            <p class="desc3i">The project for each student to test knowledge acquired</p>
+        </div>
+        </div>
+        <div><p class="aboutsee">See the whole program ></p></div>
+    </section>
+
+  <section class="speakers">
+    <h2 class="feature-speakers">Feature Speakers</h2>
+    <hr class="speaker1-hr">
+    <div class="speakerdiv"> 
+        
+    </div> 
+</section>
+
+    <footer class="partnerfooter">
+        <nav class="partnerp">
+            <div><p class="partners">Partners</p></div>
+            <hr class="speaker1m-hr">
+            <div class="footere">
+        <ul class="footeritems">
+            <li><img class="footerimg" src="img/mozilla.png" alt="Mozilla icon"></li>
+            <li><img class="footerimg" src="img/googlekorea.png" alt="Google icon"></li>
+            <li><img class="footerimg" src="img/naver.png" alt="Navar icon"></li>
+            <li><img class="footerimg" src="img/kakao.png" alt="Haumkanker icon"></li>
+            <li><img class="footerimg" src="img/airbnb.png" alt="Airbnb icon"></li>
+        </ul> </div>
+        </nav>
+        <nav class="footer2">
+            <div class="footer2-img">
+                <div class="footer-logo">
+                <img src="img/Era.png" alt="Era logo">
+                <h6 class="footer-logo-text">Era Training.</h6>
+                </div>
+            </div>
+            <h6 class="footer-desc">2022 Era Training summit<br>All Rights Reserved</h6>
+        </nav>
```

Comment:
![image](https://user-images.githubusercontent.com/60612329/164737279-860b5eac-34d8-4fab-83d0-3872c1985b71.png)
![image](https://user-images.githubusercontent.com/60612329/164737723-a889e94a-8e61-463b-8ce3-1d8544a53550.png)

- the tags should not be named ``nav `` but named ``div`` because the nav tags are used for navigations
- the header should be centralized 
- please remove excess margin at the bottom of the footer 😃